src/components/missionProfile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { JoinedMissions } from './mission';
+import { ReserveDragons } from './dragon';
+
+function Profile() {
+  const { rockets } = useSelector((store) => store.rockets);
+
+  const reservedRockets = rockets.filter((rocket) => rocket.reserved && rocket);
+
+  return (
+    <div>
+      <div className="profileContainer">
+        <div className="my_dragons">
+          <h2>My Dragons</h2>
+          <ul className="dragons_list">
+            <ReserveDragons />
+          </ul>
+        </div>
+        <div className="my_missions">
+          <h2>My Missions</h2>
+          <JoinedMissions />
+        </div>
+        <div className="my_missions ">
+          <h2>My Rockets</h2>
```

Comment:
The same applies for this My profile page please follow design guidelines. The missions part was styled according to design guidelines but the dragons and rockets were not styled according to design guideline I suggest you carefully look at the frame work design:

This is your design:

![Screenshot 2023-04-14 at 3 29 26 PM](https://user-images.githubusercontent.com/79506105/232060557-7d49995e-11c2-4066-ba37-06fd2ce67efc.png)

Please style My profile page like this:
![My Profile](https://user-images.githubusercontent.com/79506105/232060698-f4c45111-d941-4bbe-9ab8-23b1d26b6ccc.png)
Please always pay attention to detail.